src/widgets/avformatproducerwidget.cpp

```diff
@@ -1276,6 +1276,7 @@ void AvformatProducerWidget::on_rangeComboBox_activated(int index)
 {
     if (m_producer) {
         m_producer->set("color_range", index ? 2 : 1);
```

Comment:
I do not understand this line that sets "color_range". color_range is not a documented property for avformat_producer and it does not appear to use it anywhere. The "force_full_range" property is well documented, so it seems like that should be used.

Reply:
`force_full_range` should be deprecated

Reply:
That is surprising since it was just added last year:
https://github.com/mltframework/mlt/commit/0e0e526124f4392275bdeb2510e8fb086d23da8f
But I guess "color_range" is the new way.

Reply:
Oh, yeah, that is confusing. Usage of "color_range" in Shotcut was added in 2019 while "force_full_range" was added in 2022.